### DISCUSSION TITLE
Please, add support for WooCommerce PayPal Checkout Gateway

### DISCUSSION THREAD

**Discussion by iker87** [woocommerce/woocommerce#45072] [WooCommerce, Blocks, and Site Editing]
## What do you like about the Cart & Checkout blocks?
Checkout design

## What do you think is missing from the Cart & Checkout blocks?
WooCommerce PayPal Checkout Gateway support

**Comment by mikejolley** [comment]
Hello

These blocks are still in early access (via the feature plugin) and it's expected that some existing payment methods will not work with the blocks.

**That said, you can currently use PayPal Standard with the Checkout Blocks which is a gateway included with WooCommerce core.**

Currently it's up to payment method extension developers to add support for the new checkout flow offered via the cart and checkout blocks. We have some [documentation on that here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/docs/extensibility/payment-method-integration.md) and will be publicising this more in the coming months. Your best course of action right now if you are using a payment method that isn't working with the blocks is to notify the developer of the extension and ask if they can add support.

You can keep on top of the [status of the Cart & Checkout Blocks in this documentation](https://docs.woocommerce.com/document/cart-checkout-blocks-support-status/).

Thanks!

**Comment by iker87** [comment]
Hello Mike, thanks for the quick reply, you say the best action is to notify the developer... and that is what I'm doing. The plugin I'm talking about is yours

https://wordpress.org/plugins/woocommerce-gateway-paypal-express-checkout/

**Comment by mikejolley** [comment]
We have an open issue for that but no timeline for it https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/3173

**Comment by ralucaStan** [comment]
According to this https://github.com/woocommerce/woocommerce-paypal-payments/issues/211#issuecomment-1551145433 a new pre-release for the next PayPal Payments 2.0.5 is available [on GitHub](https://github.com/woocommerce/woocommerce-paypal-payments/releases/tag/2.0.5-rc1) with express payment methods support in the WooCommerce blocks.
